### PR TITLE
Fix Bastille

### PR DESCRIPTION
**Pull request by mxxtthias**

### CODE REVIEW

**Comment by Pablete1234** [comment]
What is exactly this fixing? If those lines are removed you'd keep gapples on death. It is intended that you lose them (but without dropping them on the ground) on death

**Comment by mxxtthias** [comment]
Doesn't give you items when spawning as you can see in the screenshot, that's why i deleted those lines

**Comment by mxxtthias** [comment]
Removing itemkeep and adding gapple & arrow to itemremove should fix that

**Comment by Pablete1234** [comment]
> Removing itemkeep and adding gapple & arrow to itemremove should fix that

If you add them to itemremove, players can't drop them to teammates, which is also intended

**Comment by mxxtthias** [comment]
> If you add them to itemremove, players can't drop them to teammates, which is also intended

So i think the only solution is to let them drop to the ground.

**Comment by Pablete1234** [comment]
Like it is right now, gapples drop on death, this is not intended since it makes it so you end up with stacks of gapples going arround the match if someone gets the kill on whoever is stacking them

Remove the `<clear-items/>` from blue-kit and red-kit, leave the `<clear/>` in main kit and leave everything else as-is

File: bastille/map.xml [hunk 1]
```diff
@@ -62,23 +62,21 @@
...
     <kit id="spawn-Kit">
-        <clear/>
+    	<clear/>
         <game-mode>adventure</game-mode>
         <item slot="0" unbreakable="true">stone sword</item>
         <item slot="1" unbreakable="true">bow</item>
         <item slot="2" amount="20">arrow</item>
         <item slot="8" amount="1">golden apple</item>
-		<leggings unbreakable="true">iron leggings</leggings>
+	<leggings unbreakable="true">iron leggings</leggings>
```
Comment (Pablete1234) [inline bastille/map.xml]:
```suggestion
        <leggings unbreakable="true">iron leggings</leggings>
```

File: bastille/map.xml [hunk 2]
```diff
@@ -62,23 +62,21 @@
 </respawns>
 <kits>
     <kit id="spawn-Kit">
-        <clear/>
+    	<clear/>
```
Comment (Pablete1234) [inline bastille/map.xml]:
```suggestion
        <clear/>
```

File: bastille/map.xml [hunk 3]
```diff
@@ -135,9 +133,9 @@
     <item>iron boots</item>
 </itemremove>
 <itemkeep>
-    <item>golden apple</item>
-    <item>arrow</item>
-</itemkeep>
+	<item>golden apple</item>
+	<item>arrow</item>
+</itemkeep>
```
Comment (Pablete1234) [inline bastille/map.xml]:
```suggestion
    <item>golden apple</item>
    <item>arrow</item>
</itemkeep>
```

**Review by Pablete1234 — COMMENTED** [review]
Keep the indenting as-is, everything else looks fine

**Comment by mxxtthias** [comment]
Done

**Review by Pablete1234 — APPROVED** [review]

**Comment by Pablete1234** [comment]
Thanks for the fix

**Pull request merged**